metabrainz/static/css/main.less

```diff
@@ -211,6 +211,34 @@ h1.page-title, h2.page-title {
   }
 }
 
+//team page 
+#team {
+  strong{
+    color: @green;
+  }
+  .people{
+    border-bottom: 2px solid @green;
+  }
+  .pic{
+    width: 150px;
+    padding-bottom: 10px;
+    padding-left: 10px;
+    padding-right: 10px;
+    padding-top: 5px;
+    float: left;
+  }
+  .reo{
```

Comment:
Also, if the `height` and `margin-bottom` can't be moved to be parts of `.pic` in general, then `.reo` should probably at least be renamed to something more descriptive.